src/System.Reflection.Emit.ILGeneration/dir.props

```diff
@@ -4,6 +4,5 @@
   <PropertyGroup>
     <AssemblyVersion>4.0.3.0</AssemblyVersion>
     <IsNETCoreApp>true</IsNETCoreApp>
-    <IsNETCoreAppRef>false</IsNETCoreAppRef>
```

Comment:
The Ref.Emit contracts are kind of interesting when we start to think about CoreRT, because we might be using the same NETCoreApp package. For now I'm fine with including the refs in netcoreapp package.